ndn/uri.go

```diff
@@ -240,8 +275,8 @@ func DecodeURIString(str string) *URI {
 }
 
 // GetURIType returns the type of the face URI.
-func (u *URI) GetURIType() uriType {
-	return u.uriType
+func (u *URI) GetURIType() URIType {
```

Comment:
I think it's better to change URIType back to private.
If it is public, there is no need for a getter function.